src/deadline/client/cli/_groups/config_group.py

```diff
@@ -14,52 +14,8 @@
 @_handle_error
 def cli_config():
     """
-    Manage AWS Deadline Cloud's workstation configuration.
-
-    The available AWS Deadline Cloud settings are:
-
-    defaults.aws_profile_name:
-
-        The default AWS profile to use for AWS Deadline Cloud commands. Set to '' to use the default credentials. Other settings are saved with the profile.
-
-    defaults.farm_id:
-
-        The default farm ID to use for job submissions or CLI operations.
-
-    defaults.queue_id:
-
-        The default queue ID to use for job submissions or CLI operations.
-
-    settings.storage_profile_id:
-
-        The storage profile that this workstation conforms to. It specifies
-        where shared file systems are mounted, and where named job attachments
-        should go.
-
-    defaults.job_id:
-
-        The Job ID to use by default. This gets updated by job submission so is normally the most recently submitted job.
-
-    settings.job_history_dir:
-
-        The directory in which to create new job bundles for submitting to AWS Deadline Cloud, to produce a history of job submissions.
-
-    settings.auto_accept:
-
-        Flag to automatically confirm any confirmation prompts.
-
-    settings.log_level:
-
-        Setting to change the log level. Must be one of ["ERROR", "WARNING", "INFO", "DEBUG"]
-
-    defaults.job_attachments_file_system:
-
-        Setting to determine if attachments are copied on to workers before a job executes
-        or fetched in realtime. Must be one of ["COPIED", "VIRTUAL"]
-
-    telemetry.opt_out:
-
-        Flag to specify opting out of telemetry data collection.
+    Manage AWS Deadline Cloud's workstation configuration. Config options are organized
+    and documented in config_file.SETTINGS.
```

Comment:
What steps does a user take to then figure this information from the help text? Do they have to go code spelunking? I wonder if we can avoid that. Example:

```
(deadline) bash-3.2$ deadline config
Usage: deadline config [OPTIONS] COMMAND [ARGS]...

  Manage AWS Deadline Cloud's workstation configuration. Config options are
  organized and documented in config_file.SETTINGS.

Options:
  -h, --help  Show this message and exit.

Commands:
  get   Gets an AWS Deadline Cloud workstation configuration setting.
  gui   Open the workstation configuration settings GUI.
  set   Sets an AWS Deadline Cloud workstation configuration setting.
  show  Show AWS Deadline Cloud's current workstation configuration.
```

If we do change this, the get and set command will also have to change:

```
(deadline) bash-3.2$ deadline config set --help
Usage: deadline config set [OPTIONS] SETTING_NAME VALUE

  Sets an AWS Deadline Cloud workstation configuration setting.

  For example `deadline config set defaults.farm_id <farm-id>`. Run `deadline
  config --help` to show available settings.

Options:
  -h, --help  Show this message and exit.
```

Reply:
Wrapping the documentation into `deadline config show`.

Reply:
Excerpt of sample output:
```
 > deadline config show

AWS Deadline Cloud configuration file:
   /Users/crowest/.deadline/config

deadline-cloud-monitor.path: ...
   The filesystem path to Deadline Cloud monitor, set during login process.

defaults.aws_profile_name: ...
   The AWS profile name to use by default. Set to '' to use the default
   credentials. Other settings are saved with the profile.

settings.job_history_dir: ~/.deadline/job_history/ ... (default)
   The directory in which to place the job submission history for this AWS
   profile name.

defaults.farm_id: farm-281174...
   The Farm ID to use by default.

settings.storage_profile_id:  (default)
   The storage profile that this workstation conforms to. It specifies where
   shared file systems are mounted, and where named job attachments should go.

defaults.queue_id: queue-38906...
   The Queue ID to use by default.
```